<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Mangesh-PC" failures="1" tests="6" name="Default test" time="0.217" errors="0" timestamp="14 Feb 2020 14:33:36 GMT">
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase1" time="0.0">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;uname&amp;apos; is required by @Test on method Tesstcase1 but has not been marked @Optional or defined
in C:\Users\mangesh\AppData\Local\Temp\testng-eclipse--2144885895\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'uname' is required by @Test on method Tesstcase1 but has not been marked @Optional or defined
in C:\Users\mangesh\AppData\Local\Temp\testng-eclipse--2144885895\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:290)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:359)
at org.testng.internal.Parameters.createParameters(Parameters.java:620)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Tesstcase1 -->
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase2" time="0.022"/>
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase3" time="0.004"/>
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase4" time="0.004"/>
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase5" time="0.005"/>
  <testcase classname="com.Demo.LoginTestCases" name="Tesstcase6" time="0.004"/>
</testsuite> <!-- Default test -->
